packages/react-server-dom-parcel/src/client/ReactFlightDOMClientEdge.js

```diff
@@ -30,6 +30,17 @@ import type {TemporaryReferenceSet} from 'react-client/src/ReactFlightTemporaryR
 export {createTemporaryReferenceSet} from 'react-client/src/ReactFlightTemporaryReferences';
 export type {TemporaryReferenceSet};
 
+function findSourceMapURL(filename: string, environmentName: string) {
+  const devServer = parcelRequire.meta.devServer;
```

Comment:
It's nice that this is totally encapsulated through an official API so the consumer doesn't have to do anything.